Bottom: 15392c944d5625dcdb77393363bc15b6425f4b69
Top:    311f387c95acb66531ff8ddb725dfabd487715fb
Author: student <student@ReliableEmbeddedSystems.com>
Date:   2024-01-05 20:59:20 +0000

avoid do_rootfs: Function buildhistory_get_image_installedwrite_image_manifest_to_rootfs doesn't exist

Signed-off-by: student <student@ReliableEmbeddedSystems.com>


---

diff --git a/meta/classes/buildhistory.bbclass b/meta/classes/buildhistory.bbclass
index fd53e92402..924a430746 100644
--- a/meta/classes/buildhistory.bbclass
+++ b/meta/classes/buildhistory.bbclass
@@ -692,7 +692,7 @@ python buildhistory_get_extra_sdkinfo() {
 # By using ROOTFS_POSTUNINSTALL_COMMAND we get in after uninstallation of
 # unneeded packages but before the removal of packaging files
 ROOTFS_POSTUNINSTALL_COMMAND += "buildhistory_list_installed_image"
-ROOTFS_POSTUNINSTALL_COMMAND += "buildhistory_get_image_installed"
+ROOTFS_POSTUNINSTALL_COMMAND += "buildhistory_get_image_installed "
 ROOTFS_POSTUNINSTALL_COMMAND[vardepvalueexclude] .= "| buildhistory_list_installed_image| buildhistory_get_image_installed"
 ROOTFS_POSTUNINSTALL_COMMAND[vardepsexclude] += "buildhistory_list_installed_image buildhistory_get_image_installed"
